customize.dist/src/pre-loading.css

```
@@ -20,22 +20,22 @@
-    color: #424242; /* @cp_loading-fg */
+    background-color: #0A0A0A; /* Open Paws Deep Space */
+    color: #F3F0EB; /* Open Paws Warm White */
     font-size: 1.3em;
     line-height: 120%;
     opacity: 1;
     display: flex;
     flex-flow: column;
     justify-content: center;
     align-items: center;
-    font: 20px 'Helvetica Neue', sans-serif !important;
+    font: 20px 'Space Grotesk', 'Helvetica Neue', sans-serif !important;
```

Stylelint (17.5.0)

[error] Unexpected shorthand "font" after "font-size" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)